src/main/resources/templates/faq.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    layout:decorate="layout">
+
+<head>
+    <title>FAQ</title>
+    <link th:href="@{/css/faq.css}" rel="stylesheet" />
+</head>
+
+<body>
+    <section layout:fragment="content">
+
+        <div class="faq-header">Frequently Asked Questions</div>
```

Comment:
Is there a more descriptive tag we can use here than `div`?

(hint, you've already described what it is in the class name `faq-header`)